### PR TITLE
Feature spec for change password on mobile website

### PR DESCRIPTION
**Pull request by sarithats**
Fix #6372

### CODE REVIEW

File: features/mobile/change_password.feature [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  I want to Change my password
+
+
+  Scenario: Change my password
+    Given I am signed in
+    When I go to the users edit page
+    And I fill out change password section with my password and "newsecret" and "newsecret"
+    And I press "Change password"
+    Then I should see "Password changed"
+    Then I should be on the new user session page
```
Comment (jhass) [inline features/mobile/change_password.feature]: `And`

File: features/mobile/change_password.feature [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    When I sign in with password "newsecret"
+    Then I should be on the stream page
+
+  Scenario: Attempt to change my password with invalid input
+    Given I am signed in
+    When I go to the edit user page
+    And I fill out change password section with my password and "too" and "short"
+    And I press "Change password"
+    And I should see "Password is too short"
```
Comment (jhass) [inline features/mobile/change_password.feature]: `Then`

File: features/mobile/change_password.feature [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    And I should see "Password confirmation doesn't match"
+
+  Scenario: Reset my password
+    Given a user named "Georges Abitbol" with email "forgetful@users.net"
+    Given I am on forgot password page
```
Comment (jhass) [inline features/mobile/change_password.feature]: `And`

File: features/mobile/change_password.feature [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    When I fill out forgot password form with "forgetful@users.net"
+    And I submit forgot password form
+    Then I should see "You will receive an email with instructions"
+    When I follow the "Change my password" link from the last sent email
+    When I fill out reset password form with "supersecret" and "supersecret"
```
Comment (jhass) [inline features/mobile/change_password.feature]: `And`

File: features/mobile/change_password.feature [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    And I submit reset password form
+    Then I should be on the stream page
+    And I sign out manually on the mobile website
```
Comment (jhass) [inline features/mobile/change_password.feature]: `When`

File: features/mobile/change_password.feature [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    And I sign in manually as "georges_abitbol" with password "supersecret" on the mobile website
+    Then I should be on the stream page
+
+  Scenario: Attempt to reset password with invalid password
+    Given a user named "Georges Abitbol" with email "forgetful@users.net"
+    Given I am on forgot password page
```
Comment (jhass) [inline features/mobile/change_password.feature]: `And`

File: features/mobile/change_password.feature [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    When I fill out forgot password form with "forgetful@users.net"
+    And I submit forgot password form
+    When I follow the "Change my password" link from the last sent email
+    When I fill out reset password form with "too" and "short"
```
Comment (jhass) [inline features/mobile/change_password.feature]: `And`

File: features/mobile/change_password.feature [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+    And I sign out manually on the mobile website
+    And I sign in manually as "georges_abitbol" with password "supersecret" on the mobile website
+    Then I should be on the stream page
+
+  Scenario: Attempt to reset password with invalid password
+    Given a user named "Georges Abitbol" with email "forgetful@users.net"
+    Given I am on forgot password page
+    When I fill out forgot password form with "forgetful@users.net"
+    And I submit forgot password form
+    When I follow the "Change my password" link from the last sent email
```
Comment (jhass) [inline features/mobile/change_password.feature]: `And`

**Comment by jhass** [comment]
Thanks!

**Comment by sarithats** [comment]
:+1:

**Pull request merged**